### DISCUSSION TITLE
✨ Make Windows Binaries/Zip Trusted

### DISCUSSION THREAD

**Discussion by nabsul** [dagger/dagger#12502] [Ideas]
### What are you trying to do?

I'm downloading the Dagger binaries, but Edge browser is telling me that the zip archive is untrusted.

### Why is this important to you?

It would be nice if zip/binary was signed so that Windows could trust it. I'm not sure how much effort this would involve.

### How are you currently working around this?

I have to override and ignore the warnings that edge gives me.
![image](https://github.com/dagger/dagger/assets/11143071/ea1b01d9-6320-4b29-8a83-3795305f4581)

**Comment by jedevc** [comment]
I *think* the binaries downloadable through winget (documented as the primary windows method in https://docs.dagger.io/getting-started/installation) shouldn't have this problem.
